docs/concept-exercises.md

```diff
@@ -51,7 +51,9 @@ An exercise has the following files. In the browser they will show at the releva
 
 ### `.docs/introduction.md`
 
-This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms that the student can Google if they so want, is preferable over including any code samples or an extensive description. For example we might describe a string as a "Sequence of Unicode characters" or a "series of bytes" or "an object". Unless the student needs to understand the details of what those mean to be able to solve the exercise we should not give more info in this introduction - instead allowing the student to Google, ignore, or map their existing knowledge.
+This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise, and provide a short, concise introduction to the concept(s). The aim is to give the student just enough context to figure things out themselves and solve the exercise, as research has shown that self-discovery is the most effective learning experience. Mentioning technical terms, that the student can Google if they so want, is preferable over an extensive description. The exception to this is syntax, which we don't want the student to need Google, so code samples should be included whenever syntax is introduced.
```

Comment:
Avoid too many commas. Consider: "...the student should learn from the exercise and provide a short, concise..."

Reply:
@fireproofsocks Could you use GitHub's "Insert a suggestion" feature? That makes it much easier for me to apply any suggested changes.

Reply:
I think it's important that in making the sentences read better we don't lose the "why". So for example, I want to make sure we keep saying that we *do* want students to Google the "skills" bits but also clearly say we *don't* want them to Google syntax. Those are design decisions and I want this to communicate those design decisions so maintainers can understand the framework of our thinking, rather than just having rules to follow.

It might even be worth splitting this into a paragraph that explains the mental framework, and then bullet points that cover how those rules would then be applied.

Reply:
To put it another way, I'm a huge fan of being **more** verbose in these docs as clarity is everything, as these are effectively specs.

Reply:
I'm all for stating things _clearly_.  Communicating unambiguously can be hard...

Also... it reads weird to me to peg "Google" as the de-facto web search.

Reply:
How would you feel about something like the following:

```
This file contains an introduction to the concept. It should be explicit about what the student should learn from the exercise and provide a short introduction with enough detail for the student to complete the exercise. The aim is to give the student just enough context to figure out the solution themselves, as research has shown that self-discovery is the most effective learning experience.  Using the proper technical terms in the descriptions will be helpful for conducting web searches. An example of the proper code syntax should always be included: students should not need to search the web for examples of syntax.

This file should:
- Avoid extensive descriptions
- Mention technical terms that will aid the student in web searches
- Provide examples of code syntax

For example, the introduction to a "strings" exercise might describe a string as a "Sequence of Unicode characters" or a "series of bytes" or simply as "an object". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation along with an example of the proper code syntax should be sufficient information for the student to solve the exercise.
```

Reply:
I'm not entirely sure. I don't really like the bullet points _and_ the description describing the same things. That to me _is_ duplication I'd like to avoid :) How about this:

> This file contains an introduction to the concept. It should make the exercise's learning goals explicit and provide a short introduction with enough detail for the student to complete the exercise. The aim is to give the student just enough context to figure out the solution themselves, as research has shown that self-discovery is the most effective learning experience. Using the proper technical terms in the descriptions will be helpful if the student wants to search for more information. If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
> 
> As an example, the introduction to a "strings" exercise might describe a string as just a "Sequence of Unicode characters" or a "series of bytes". Unless the student needs to understand more nuanced details in order to solve the exercise, this type of brief explanation (along with an example of its syntax) should be sufficient information for the student to solve the exercise.

@fireproofsocks @iHiD what do you think of the above suggestion?

Reply:
I think that's much better.

Reply:
Looks good.  The bullet points were, by the way, were in response to @iHiD 's response.

Reply:
I see.

Reply:
@fireproofsocks Could you approve the PR if it is sufficiently to your liking? I cannot merge as you've requested changes :)